tests/tests_pytorch/strategies/test_fsdp.py

```diff
@@ -11,8 +11,8 @@
 from pytorch_lightning import Trainer
 from pytorch_lightning.callbacks import ModelCheckpoint
 from pytorch_lightning.demos.boring_classes import BoringModel
-from pytorch_lightning.plugins.precision.fsdp_native_native_amp import FullyShardedNativeNativeMixedPrecisionPlugin
-from pytorch_lightning.strategies import DDPFullyShardedNativeStrategy
+from pytorch_lightning.plugins.precision.fsdp_amp import FSDPMixedPrecisionPlugin
```

Comment:
fyi, in fabric the module is called fsdp

Reply:
Good catch. Changed